frontend/ui/elements/screen_eink_opt_menu_table.lua

```diff
@@ -23,3 +23,10 @@ return {
 
     },
 }
+
+-- TODO reactivate if someone reverse engineers Android E Ink stuff
+if Device:isAndroid() then
+    table.insert(eink_settings_table.sub_item_table, 1, require("ui/elements/refresh_menu_table"))
```

Comment:
Shouldn't this be` if not Device:isAndroid() then` ?

Reply:
Quite so! :-D

Reply:
(I quickly tested on not Android and forgot to change it back.)